pkg/ui/Makefile

```diff
@@ -62,7 +62,7 @@ test-watch: $(DLLS)
 dist/%.dll.js %-manifest.json: webpack.%.js | protos
 	$(WEBPACK) -p --config $<
 
-$(GOBINDATA_TARGET): webpack.app.js $(DLLS) $(shell find src)
+$(GOBINDATA_TARGET): webpack.app.js $(DLLS) $(shell find src) $(shell find styl)
```

Comment:
nit: `$(shell find src styl)`

(Every shell command that appears in a prerequisite has to be executed on every invocation of `make`, so it's good to limit the number as much as possible.)

Reply:
:+1: done